code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -720,6 +774,113 @@ to_chat will check for valid clients itself already so no need to double check f
 		if(xeno_job.total_positions < (-difference + xeno_job.current_positions))
 			xeno_job.set_job_positions(-difference + xeno_job.current_positions)
 
+
+///Handles the timer when all silos are destroyed
+/datum/hive_status/proc/handle_silo_death_timer()
+	return
+
+/datum/hive_status/normal/handle_silo_death_timer()
+	if(!isdistressgamemode(SSticker.mode))
```

Comment:
i dunno why you are moving that, but since you are, replace that by gamemode flag check rather than that disgusting istype check

Reply:
changed to larva spawn flag